internal-ui/src/components/DashboardComponent.tsx

```diff
@@ -1,23 +1,76 @@
-import React from 'react';
-import { CalendarMonth } from '@patternfly/react-core';
-import { Checkbox } from '@patternfly/react-core';
+import React, { useEffect, useState } from 'react';
 import {
+  CalendarMonth,
+  Checkbox,
   Card,
-  CardBody,
-  CardTitle,
-  CardHeader,
+  Grid,
+  GridItem,
 } from '@patternfly/react-core';
-import { useState } from 'react';
-
+import {
+  ChartDonut,
+  ChartPie,
+  ChartArea,
+  ChartScatter,
+  ChartThemeColor,
+  ChartVoronoiContainer,
+  ChartBar
+} from '@patternfly/react-charts';
+import { getMessagesInRange } from '../services/messages';
+import { Message } from '../Types';
 
-export const SelectFilters = () => {
+export const DashboardComponent = () => {
```

Comment:
Not sure if it's just me, but here's how the page looks. Did you forget to push maybe some styling changes?

Since the date/time picker is broken, the stats are 0
![Peek 2024-08-06 13-44](https://github.com/user-attachments/assets/b3716ac4-878d-4bbd-8361-dc3f9beef447)